src/nu/validator/checker/schematronequiv/Assertions.java

```diff
@@ -3157,6 +3157,17 @@ else if ("bdo" == localName && atts.getIndex("", "dir") < 0) {
                             + " \u201Cprerender\u201D, or"
                             + " \u201Cstylesheet\u201D.");
                 }
+                if (atts.getIndex("", "blocking") > -1
+                        && atts.getValue("", "blocking").equalsIgnoreCase("render")
```

Comment:
Per https://html.spec.whatwg.org/multipage/semantics.html#attr-link-blocking the spec states:
> The blocking attribute is a [blocking attribute](https://html.spec.whatwg.org/multipage/urls-and-fetching.html#blocking-attribute). It is used by link types [modulepreload](https://html.spec.whatwg.org/multipage/links.html#link-type-modulepreload), [preload](https://html.spec.whatwg.org/multipage/links.html#link-type-preload) and [stylesheet](https://html.spec.whatwg.org/multipage/links.html#link-type-stylesheet), and it must only be specified on link elements that have a [rel](https://html.spec.whatwg.org/multipage/semantics.html#attr-link-rel) attribute containing one of those keywords.

That means we don’t need to check the value of the `blocking` attribute here, right? — Because the spec states the attribute itself (regardless of its value) must only be used with elements that also have one of the given `rel` values.

Reply:
You are right. I removed this part of the condition and edited message.